# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1252191759 14400
#      Sat Sep 05 19:02:39 2009 -0400
# Node ID b0b82ab654f6cf71db9e93aaca2bac897fbee8ca
# Parent  e381f80a5f7ae776393a66e141467dded5035ab0
txt-eng-ft.cc (ft_manager::do_get_font): remove spurious assignment used for debugging

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2009-09-05  John W. Eaton  <jwe@octave.org>
+
+	* txt-eng-ft.cc (ft_manager::do_get_font): Remove spurious
+	assignment usd for debugging.
+
 2009-09-04  John W. Eaton  <jwe@octave.org>
 
 	* txt-eng-ft.cc (ft_manager::do_get_font, ft_render::set_font,
 	ft_render::set_mode, ft_render::visit, ft_render::set_color):
 	Issue warnings instead of errors for non-fatal problems.
 	(ft_render::visit): If face is null, then return without warning.
 	* genprops.awk: Also generate const std::string& and const char*
 	versions of the get member functions.
diff --git a/src/txt-eng-ft.cc b/src/txt-eng-ft.cc
--- a/src/txt-eng-ft.cc
+++ b/src/txt-eng-ft.cc
@@ -135,17 +135,16 @@ private:
 
 	  if (FcConfigSubstitute (0, pat, FcMatchPattern))
 	    {
 	      FcResult res;
 	      FcPattern *match;
 
 	      FcDefaultSubstitute (pat);
 	      match = FcFontMatch (0, pat, &res);
-	      match = 0;
 	      
 	      if (match && res != FcResultNoMatch)
 		{
 		  unsigned char *tmp;
 
 		  FcPatternGetString (match, FC_FILE, 0, &tmp);
 		  file = reinterpret_cast<char*> (tmp);
 		}
